dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/data/upload/DataUploadRunnable.kt

```diff
@@ -73,10 +74,9 @@ internal class DataUploadRunnable(
     }
 
     private fun consumeBatch(batch: Batch) {
-        val batchId = batch.id
-        sdkLogger.i("Sending batch $batchId")
         val status = dataUploader.upload(batch.data)
-        status.logStatus(dataUploader.javaClass.simpleName, batch.data.size)
+        status.logStatus(dataUploader.javaClass.simpleName, batch.data.size, devLogger, false)
```

Comment:
I think this will duplicate the logs in Logcat if we have: 
`BuildConfig.LOGCAT_ENABLED = true` and `Dadadog.setVerbosity(Log.INFO)`

Reply:
`BuildConfig.LOGCAT_ENABLED = true` will only be there for us, so that shouldn't be an issue

Reply:
Yes, am aware it will only be for us was just a heads up. This can be miss leading for us in the Logcat.